scripts/setup.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# =============================================================================
+# DEPENDENCY VERSIONS
+# =============================================================================
+NVM_VERSION="v0.40.3"
+NODE_VERSION="22"
+YARN_VERSION="4.6.0"
+SOLANA_VERSION="v1.17.25"
+ANCHOR_VERSION="v0.29.0"
+RUST_VERSION_FOR_ANCHOR="1.76.0"    # Required for building Anchor v0.29.0
```

Comment:
Could you use 1.78.0 ?

`yarn build` should build everything. It uses `1.78.0 for contract build`. (no need to switch default, we can use Rust 1.85.1 as default because package.json has the following setting)
https://github.com/orca-so/whirlpools/blob/main/programs/whirlpool/package.json#L5